### PR TITLE
Update adding HillCore + ability to generate hills

### PR DESCRIPTION
**Pull request by mykelp**

### CODE REVIEW

File: contracts/HillCore.sol [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  function transferTo( 
+    address to, uint256 assetId, bytes userData, bytes operatorData
+  ) 
+    public
+  {
+    return transfer(to, assetId, userData, operatorData);
+  }
+}
```
Comment (pfletcherhill) [inline contracts/HillCore.sol]: Add new line at end of file

Comment (mykelp) [reply contracts/HillCore.sol]: fixed

File: contracts/HillCore.sol [hunk 2]
```diff
@@ -0,0 +1,36 @@
+pragma solidity 0.4.19;
+
+import './StandardAssetRegistry.sol';
+
+contract HillCore is StandardAssetRegistry {
+
+  function HillCore() public {
+    _name = "Crypto Hills";
+    _symbol = "CH";
```
Comment (pfletcherhill) [inline contracts/HillCore.sol]: `_symbol = "HILL"`

File: package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "git+https://github.com/hillstreetlabs/registry.git"
   },
-  "author": "The Hill St Labs Team",
+  "author": "The Hill Street Labs",
```
Comment (pfletcherhill) [inline package.json]: Just "Hill Street Labs"

**Pull request merged**